### PR TITLE
more text reading tests

### PR DESCRIPTION
**Pull request by alicekwak**
added some tests

### CODE REVIEW

File: automates/text_reading/src/main/resources/application.conf [hunk 1]
```diff
@@ -94,7 +94,7 @@ apps {
 
 alignment {
   alignerType = "pairwisew2v"
-  w2vPath = "/local/path/to/vectors.txt" // a word embeddings file (e.g., GloVe) with this header: <len_of_vocab><space><embedding_size>, e.g., "6B 50" (no quotes); use of different sets of embeddings will require adjusting alignment similarity thresholds, e.g., commentTextAlignmentScoreThreshold
+  w2vPath = "/Users/alicekwak/repos/automates/automates/text_reading/vectors.txt" // a word embeddings file (e.g., GloVe) with this header: <len_of_vocab><space><embedding_size>, e.g., "6B 50" (no quotes); use of different sets of embeddings will require adjusting alignment similarity thresholds, e.g., commentTextAlignmentScoreThreshold
```
Comment (adarshp) [inline automates/text_reading/src/main/resources/application.conf:97]: @alicekwak - The version of the config file in the repo should not have a user-specific path...

Comment (maxaalexeeva) [reply automates/text_reading/src/main/resources/application.conf:97]: @adarshp, thank you! This is me and Alice practising opening pull requests (this is a draft with more changes coming) and we will make sure to keep the conf file free of use-specific paths.

Comment (adarshp) [reply automates/text_reading/src/main/resources/application.conf:97]: @maxaalexeeva got it, no worries :)

**Comment by maxaalexeeva** [comment]
Hi, @alicekwak, thanks! I am in the process of merging this branch into my conjunctions/other fixes branch (there are a couple of more things I want to fix, but almost done). I will close this pr after I'm done.

**Comment by maxaalexeeva** [comment]
Closing this pr because these changes have been merged with PR #187.

**Pull request closed without merging**